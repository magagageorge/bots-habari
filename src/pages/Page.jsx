import React from "react";

class Page extends React.Component {
    render() {
        return (<div className="container-fluid animated fadeIn">
            <div className="row mx-0 w-100">
                <div className="col-12 d-flex-justify-content-center">
                    <p>Other  Page Content Goes here.....</p>
                </div>
            </div>
        </div>)
    }
}

export default Page;