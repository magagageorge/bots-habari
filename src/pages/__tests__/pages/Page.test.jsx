
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Page from '../../Page';


describe('pages/Page test', () => {
  it('pages/Page should match snapshot', () => {
    const component = renderer.create(<Page
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
