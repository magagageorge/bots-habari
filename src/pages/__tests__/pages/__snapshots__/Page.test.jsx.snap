// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/Page test pages/Page should match snapshot 1`] = `
<div
  className="container-fluid animated fadeIn"
>
  <div
    className="row mx-0 w-100"
  >
    <div
      className="col-12 d-flex-justify-content-center"
    >
      <p>
        Other  Page Content Goes here.....
      </p>
    </div>
  </div>
</div>
`;
